src/api/Alert/types.ts

```diff
@@ -165,6 +166,7 @@ export interface GetCasesResponse {
     unconfirmed: number;
     confirmed: number;
     dismissed: number;
+    in_progress: number;
```

Comment:
@yigitakbulut since you put in_progress here, could we put in_progress intead of in-progress everywhere? we use it in the frontend that way. If you feel it does not make sense I can do it differently